domain-model/src/main/java/md/maib/retail/Model/Campaign/Campaign.java

```diff
@@ -0,0 +1,32 @@
+package md.maib.retail.Model.Campaign;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import md.maib.retail.Model.Conditions.Rule;
+import md.maib.retail.Model.Effects.LoyaltyEventType;
+import org.threeten.extra.Interval;
+import java.util.Collection;
+import java.util.Map;
+import java.util.UUID;
+@Getter
```

Comment:
Add empty line after imports

Reply:
After I've done it, do I need to create a new commit and push?
or edit it only in IDE